src/anonfile/__init__.py

```diff
@@ -77,21 +79,25 @@ def main():
         if args.command == 'preview':
             for url in args.url:
                 preview = anon.preview(url)
-                values = ['online' if preview.status else 'offline', preview.file_path.name, preview.url.geturl(), preview.ddl.geturl(), preview.id, f"{preview.size_readable}"]
+                response = {
+                    'Status': 'online' if preview.status else 'offline',
+                    'File Path': preview.file_path.name,
+                    'URL': preview.url.geturl(),
+                    'DDL': preview.ddl.geturl(),
+                    'ID': preview.id,
+                    'Size': preview.size_readable,
+                }
 
-                if args.verbose:
-                    __print_dict(dict(zip(['Status', 'File Path', 'URL', 'DDL', 'ID', 'Size'], values)))
-                else:
-                    print(','.join(values))
+                print(json.dumps(response, indent=4) if args.verbose else ','.join(response.values))
 
         if args.command == 'download':
             for url in (args.url or __from_file(args.batch_file)):
                 download = lambda url: anon.download(url, args.path, progressbar=args.verbose, enable_logging=args.logging)
 
                 if args.check and anon.preview(url, args.path).file_path.exists():
-                    print(f"\033[33mWarning:\033[0m A file with the same name already exists in {str(args.path)!r}.")
-                    choice = input("Proceed with download? [Y/n] ")
-                    if choice == '' or strtobool(choice):
+                    print(f"Warning: A file with the same name already exists in {str(args.path)!r}.")
```

Comment:
This looks good. I probably wouldn't have thought of this. I thought it just overwrote the original. Maybe I'm wrong, but this is a good solution to the possible conflict. Thanks!